src/scenegraph/Label3D.cpp

```diff
@@ -54,7 +54,7 @@ void Label3D::SetText(const std::string &text)
 		// Most noticeably, this means text consisting of entirely Cyrillic
 		// or Chinese characters will vanish when rendered on a Label3D.
 		if (m_geometry->IsEmpty()) { return; }
-		
+
```

Comment:
I'm about to do a whole codebase one just for whitespace so I'm letting these ones slide ;)